composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "fzaninotto/faker" : "^1.6"
+        "fakerphp/faker" : "^1.11"
```

Comment:
Needs to be `^1.9.1`. This is very important to allow co-installation with the old package.

Reply:
Done